AntennaTracker/GCS_Mavlink.cpp

```diff
@@ -98,6 +99,64 @@ void Tracker::send_nav_controller_output(mavlink_channel_t chan)
         0);
 }
 
+/*
+  send PID tuning message
+ */
+void Tracker::send_pid_tuning(mavlink_channel_t chan)
+{
+    const AP_Logger::PID_Info *pid_info;
+
+    // Pitch PID
+    if (g.gcs_pid_mask & 1) {
+        pid_info = &g.pidPitch2Srv.get_pid_info();
+        mavlink_msg_pid_tuning_send(chan, PID_TUNING_PITCH,
+                                    pid_info->desired,
+                                    pid_info->actual,
+                                    pid_info->FF,
+                                    pid_info->P,
+                                    pid_info->I,
+                                    pid_info->D);
+        if (!HAVE_PAYLOAD_SPACE(chan, PID_TUNING)) {
+            return;
+        }
+    }
+
+    // Yaw PID
+    if (g.gcs_pid_mask & 2) {
+        pid_info = &g.pidYaw2Srv.get_pid_info();
+        mavlink_msg_pid_tuning_send(chan, PID_TUNING_YAW,
+                                    pid_info->desired,
+                                    pid_info->actual,
+                                    pid_info->FF,
+                                    pid_info->P,
+                                    pid_info->I,
+                                    pid_info->D);
+        if (!HAVE_PAYLOAD_SPACE(chan, PID_TUNING)) {
+            return;
+        }
+    }
+}
+
+// send position tracker is using
+void Tracker::send_global_position(mavlink_channel_t chan)
+{
+    AP_AHRS &ahrs = AP::ahrs();
+
+    Vector3f vel;
+    ahrs.get_velocity_NED(vel);
+
+    mavlink_msg_global_position_int_send(
```

Comment:
This is so we can sent the current location used by tracker, due to the start lat and long parameters it may not be the same as the ahrs location used by default. I think we could set the ahrs origin to make it correct, but then i think it will use inertial nav so we might get drift. if the start lat and long are set i think its safe to assume were not going anywhere.

Reply:
now sends zero velocity if using start lat and log